internal/core/application/feeder/types.go

```diff
@@ -0,0 +1,13 @@
+package feeder
+
+type marketInfo struct {
+	baseAsset  string
+	quoteAsset string
+}
+
+func (i marketInfo) GetBaseAsset() string {
+	return i.baseAsset
+}
+func (i marketInfo) GetQuoteAsset() string {
+	return i.quoteAsset
+}
```

Comment:
we can simplify this a little bit like:

```suggestion
type marketInfo domain.Market

func (i marketInfo) GetBaseAsset() string {
	return i.BaseAsset
}
func (i marketInfo) GetQuoteAsset() string {
	return i.QuoteAsset
}
```

Reply:
Done